src/app/tests/suites/certification/Test_TC_DISHALM_3_1.yaml

```diff
@@ -11,95 +11,148 @@
-          attribute"
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Operate device to set State attribute bit 0 (Inflow) to 1"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "TH reads from the DUT the State attribute"
       PICS: DISHALM.S.A0002
-      verification: |
-          Bit 0 of response SHALL be 0 if bit 0 of DISHALM.S.A0001(Latch) is 0
-          Bit 0 of response SHALL be 1 if bit 0 of DISHALM.S.A0001(Latch) is 1
-      disabled: true
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          constraints:
+              hasMasksSet: [0x1]
+
+    - label: "Operate device to set State attribute bit 0 (Inflow) to 0"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "TH reads from the DUT the State attribute"
+      PICS: " !DISHALM.S.A0001 && DISHALM.S.A0002 "
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 0
+
+    - label: "TH reads from the DUT the State attribute"
+      PICS: DISHALM.S.A0001 && DISHALM.S.A0002
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          constraints:
+              hasMasksSet: [0x1]
 
     - label:
-          "Step 3a: TH sends command Reset to DUT with bit 0 (Inflow) of Alarms
-          set to 1"
-      PICS: DISHALM.S.F00 && DISHALM.S.C00.Rsp
+          "TH sends command Reset to DUT with bit 0 (Inflow) of Alarms set to 1"
       verification: |
-
-      disabled: true
-
-    - label:
-          "Step 3b: After a few seconds, TH reads from the DUT the State
-          attribute"
+          ./chip-tool dishwasheralarm reset 1 1 1
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "TH reads from the DUT the State attribute"
       PICS: DISHALM.S.F00 && DISHALM.S.A0002
-      verification: |
-          Bit 0 of response SHALL be 0
-      disabled: true
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 0
 
-    - label: "Step 4a: Set up subscription to Notify event"
+    - label: "Set up subscription to Notify event"
```

Comment:
This ordering does not seem to match the test plan.  If that's purposeful, please document why it's being done.